engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFBO.java

```diff
@@ -114,20 +112,13 @@ public StateChange getDefaultInstance() {
     }
 
     private int fetchTextureId() {
-
         // TODO: make checks to verify the FBOs has the requested buffer;
-
         if (inputFbo != null) {
-
             switch (textureType) {
-
                 case ColorTexture:
                     return inputFbo.colorBufferTextureId;
 
-                case DepthRenderBuffer:
-                    return inputFbo.depthStencilRboId;
-
-                case DepthTexture:
+                case DepthStencilTexture:
```

Comment:
Giving a little bit more information, even if it's not strictly required, doesn't hurt.